packages/react-components/src/components/styles/ChatMessageComponent.styles.ts

```diff
@@ -126,7 +126,8 @@ export const chatMessageMenuStyle = mergeStyles({
   minWidth: '8.5rem',
   height: 'max-content',
   cursor: 'pointer',
-  overflow: 'hidden'
+  overflow: 'hidden',
+  '-webkit-tap-highlight-color': 'rgba(255, 255, 255, 0)' // Disable tap highlight on Android
```

Comment:
It might be better to use `transparent` instead of `rgba(255, 255, 255, 0)` because of compatibility(with older browsers) and readability